src/lib/containers/TotalQueryResults.tsx

```diff
@@ -2,7 +2,7 @@ import { cloneDeep } from 'lodash-es'
 import React, { FunctionComponent, useState } from 'react'
 import { Button } from 'react-bootstrap'
 import useDeepCompareEffect from 'use-deep-compare-effect'
-import { SynapseClient, SynapseConstants } from '../'
+import { SynapseClient, SynapseConstants } from '../utils'
```

Comment:
The previous import resolves to `src/lib/index.ts` which pulls in modules and SCSS for the webpack build that we don't want in the UMD build